docs/fundamentals/code-analysis/rules/usage-warnings.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Usage Warnings (code analysis)
```

Comment:
Can we change the content and title for all category pages to use the term "Rules" instead of "Warnings"? Similar to the changes done in https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/7185

Reply:
Yes, sorry that I missed those changes you made in the VS repo.